changelog.md

```diff
@@ -1,45 +1,76 @@
-* fix: read gas limit from smart contract by @lumtis in https://github.com/zeta-chain/node/pull/1277
-* fix(`fungible`): add CLI command to query system contract by @lumtis in https://github.com/zeta-chain/node/pull/1252
-* fix(`cmd`): add notice when using `--ledger` with Ethereum HD path by @lumtis in https://github.com/zeta-chain/node/pull/1285
-* fix: gosec issues by @lumtis in https://github.com/zeta-chain/node/pull/1290
-* refactor: remove duplicate funtion name IsEthereum by @lukema95 in https://github.com/zeta-chain/node/pull/1279
-* fix: query outtx tracker by chain using prefixed store by @ws4charlie in https://github.com/zeta-chain/node/pull/1283
-* refactor: skip gas stability pool funding when gasLimit is equal gasUsed by @lukema95 in https://github.com/zeta-chain/node/pull/1289
-* fix: minor fixes to stateful upgrade by @kevinssgh in https://github.com/zeta-chain/node/pull/1280
-* ci: add mainnet builds to goreleaser by @CharlieMc0 in https://github.com/zeta-chain/node/pull/1302
-* feat(`fungible`): add query command to get all gas stability pool balances by @lumtis in https://github.com/zeta-chain/node/pull/1247
-* feat: tss funds migration by @kingpinXD in https://github.com/zeta-chain/node/pull/1143
-* fix(`gas-payment`): remove check `gasObtained == outTxGasFee` by @lumtis in https://github.com/zeta-chain/node/pull/1304
-* chore: sync from mockmain  by @brewmaster012 in https://github.com/zeta-chain/node/pull/1265
-* chore: increment handler version by @kingpinXD in https://github.com/zeta-chain/node/pull/1307
-* fix: begin blocker for mock mainnet by @kingpinXD in https://github.com/zeta-chain/node/pull/1308
+### Features
+* [1137](https://github.com/zeta-chain/node/pull/1137) - external stress testing
+* [1205](https://github.com/zeta-chain/node/pull/1205) - allow setting liquidity cap for ZRC20
+* [1260](https://github.com/zeta-chain/node/pull/1260) - add ability to update gas limit
+* [1263](https://github.com/zeta-chain/node/pull/1263) - Bitcoin block header and merkle proof
+* [1247](https://github.com/zeta-chain/node/pull/1247) - add query command to get all gas stability pool balances
+* [1143](https://github.com/zeta-chain/node/pull/1143) - tss funds migration capability
+
+### Fixes
+
+* [1195](https://github.com/zeta-chain/node/pull/1195) - added upgrade name, and allow download. allows to test release
+* [1153](https://github.com/zeta-chain/node/pull/1153) - address `cosmos-gosec` lint issues
+* [1128](https://github.com/zeta-chain/node/pull/1228) - adding namespaces back in rpc
+* [1245](https://github.com/zeta-chain/node/pull/1245) - set unique index for generate cctx
+* [1250](https://github.com/zeta-chain/node/pull/1250) - remove error return in `IsAuthorized`
+* [1261](https://github.com/zeta-chain/node/pull/1261) - Ethereum comparaison checksum/non-checksum format
+* [1264](https://github.com/zeta-chain/node/pull/1264) - Blame index update by @kevinssgh
```

Comment:
We can decide fo or against keeping github handles , but lets keep it uniform ? 
In my opinion the names are redundant as the prs are already linked

Reply:
Yes, I think it's clearer removing handles, this is a typo

Reply:
[b731d6c](https://github.com/zeta-chain/node/pull/1390/commits/b731d6cb9deff879041346cd66fbf5c7b37bd95c)